packages/itwinui-react/src/core/List/List.tsx

```diff
@@ -12,20 +12,26 @@ import type {
 import '@itwin/itwinui-css/css/menu.css';
 
 export const List = React.forwardRef((props, ref) => {
-  const { as: Element = 'ul', className, ...rest } = props;
+  const { as: Element = 'ul', className, role = 'list', ...rest } = props;
```

Comment:
undo changes in this file. `role` prop should automatically be respected as it will be part of `...rest`.

Reply:
Fixed